gateway/ln-gateway/proto/gatewaylnrpc.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto3";
```

Comment:
I wonder if we can just emulate LND with the CLN plugin such that we just need one impl in the gateway daemon?